Src/Support/Google.Apis.Auth.AspNetCore3.IntegrationTests/Views/Home/ShowTokens.cshtml

```diff
@@ -9,23 +11,27 @@
 <html>
 <head>
     <meta name="viewport" content="width=device-width" />
-    <title>Integreation tests - Show Tokens</title>
+    <title>Google.Apis.Auth.AspNetCore3 sample - OAuth Tokens</title>
 </head>
 <body>
-    <h1>Integration tests - Show Tokens</h1>
-    <ul>
-        @foreach (var token in @Model)
-        {
-            <li>@token</li>
-        }
-    </ul>
+    <h2>Google.Apis.Auth.AspNetCore3 sample - OAuth Tokens</h2>
     @if (@Model.Count > 0)
     {
-        <p>Test passed</p>
+        <p>
+            @User.FindFirst(ClaimTypes.GivenName)?.Value here is information about the OAuth tokens
```

Comment:
(And again - I'm only continuing to add these so we don't miss any. If you decide to leave things where they are, ignore all of these.)